### ISSUE TITLE
Controls not showing

### ISSUE THREAD

**Issue by miesjel12** [kalkih/mini-media-player#374]
Hi all,

I've have a strange issue (at least for me) and its quite annoying. The control buttons for volume, power etc are not shown after i've updated to the latest version. The problem consists in multiple themes, even the default one. The buttons do work, i just cant see them. But if i press in the area where to power button should be, the device reacts as expected. 

I've tried adding the hide controls and then set it as false but it didn't fall for that trick!

![naamloos](https://user-images.githubusercontent.com/40541145/89291672-3d7a1380-d65b-11ea-940c-ed693660f1b0.png)

The code:
```artwork: cover
entity: media_player.pioneer_vsx_1131_dfb820
icon: 'mdi:radio-tower'
info: short
hide:
  controls: false
name: Pioneer woonkamer
power: true
shortcuts:
  buttons:
    - id: script.1585849558340
      name: Radio 538
      type: script
    - id: script.1587216594173
      name: Radio 538 GDC
      type: script
    - id: script.1587215998491
      name: Radio 10 Non stop
      type: script
    - id: script.1587216320761
      name: MNM Hits
      type: script
  columns: 2
type: 'custom:mini-media-player'
volume_stateless: true
```

**Comment by kalkih** [comment]
Hello, update the card if you haven't in a while, see https://github.com/kalkih/mini-media-player/issues/297 & https://github.com/kalkih/mini-media-player/issues/302 for more information.
Make sure to clear cache, reload resources, try incognito e.t.c. if you're still having issues.

**Comment by miesjel12** [comment]
Hi,

I've checked those before reporting this. I should have added this information, apologies! 

The card is updated to the latest version: V1.10.1. I've reinstalled the add on, i've cleared caches, i've used other devices, i tried Icognito but the problem is still there. I just rebooted the whole host of Home assistant but that did not help.

**Comment by kalkih** [comment]
Is the issue present in all browsers? Which ones did you try?
What HA version are you running?

**Comment by miesjel12** [comment]
Present in at least Google Chrome, Edge and Firefox (all updated to latest version). Home assistant is also up to date, version 0.113.3

**Comment by kalkih** [comment]
I am not able to reproduce this, my best guess is that you're loading an old version of the card somehow, make sure your resource reference is up to date and matching what's stated in HACS (if installed that way).
I would recommend uninstalling the card completely, make sure the UI is broken / the mini-media-player cards doesn't show, and then install it again.

**Comment by miesjel12** [comment]
Found the issue, I think!

A noticed an older version on my custom_ui file while I also had one installed via HACS. I've removed both and installed it fresh via HACS. Now it works as expected, on one dashboard that is. On another lovelace dashboard i still have the same issue but i will fix that somehow. 

Thanks for assisting and pointing me in the right direction and thanks for making an awesome plugin!
